scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -95,62 +153,50 @@ trait AndroidAppModule extends JavaModule {
    * [[https://developer.android.com/tools/d8 d8 Documentation]]
    */
   def androidJar: T[PathRef] = Task {
-    val jarFile: os.Path = T.dest / "app.jar"
+    val jarFile: os.Path = Task.dest / "app.jar"
 
     os.call(
       Seq(
-        androidSdkModule().d8Path().path.toString, // Call d8 tool
+        androidSdkModule().d8Path().path.toString,
         "--output",
-        jarFile.toString, // Output JAR file
-        "--no-desugaring" // Disable desugaring
+        jarFile.toString,
+        "--no-desugaring"
```

Comment:
desugaring is a configurable parameter in the build setup in Gradle, so maybe it makes sense to make it configurable here as well

Reply:
Yes we can i will update to desugaring